README.md

```diff
@@ -10,8 +10,42 @@ Sails.js/Waterline adapter for MongoDB.
 > As an adapter, this module implements a set of declarative interfaces, conventions, and best-practices for integrating with Mongo databases.
 > Strict adherence to an adapter specification enables the (re)use of built-in generic test suites, standardized documentation, reasonable expectations around the API for your users, and overall, a more pleasant development experience for everyone.
 
+# BREAKING CHANGES
 
-## Installation
+## MongoDB Driver
+From `sails-mongo` version 2.0.0 and above, the adapter uses a new mongodb driver. These driver changes the way it handles connections,
+and implements the concept of [MongoClient](http://mongodb.github.io/node-mongodb-native/3.2/api/MongoClient.html).
+
+Because of that, `manager` now returns MongoClient, instead of just a connection.
+
+This enables a lot more flexibility and allows the use of the latest MongoDb improvements, like (ClientSession)[http://mongodb.github.io/node-mongodb-native/3.2/api/ClientSession.html],
+and with it, transactions, change streams, and other new features.
+
+## Access to Database object
+If you need to get the database, you have to call the [`db`](http://mongodb.github.io/node-mongodb-native/3.2/api/MongoClient.html#db) method on the manager (MongoClient).
```

Comment:
is it a method (`.db()`) or a property (`.db`)?

Also I think it would be good to clarify this a bit more:

> If you need to get the database, you have to call

i.e. if you need to get the <<insert mongo's specific term of art for whatever this is called here>>

Reply:
Added much better explanations with example